modelarmor/api/ModelArmor.Samples/CreateTemplateWithBasicSdp.cs

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+// [START modelarmor_create_template_with_basic_sdp]
+using System;
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.ModelArmor.V1;
+
+public class CreateTemplateWithBasicSdpSample
+{
+    public Template CreateTemplateWithBasicSdp(
+        string projectId = "my-project",
+        string locationId = "us-central1",
+        string templateId = "my-template")
+    {
+        // Construct the API endpoint URL.
+        ModelArmorClientBuilder clientBuilder = new ModelArmorClientBuilder
+        {
+            Endpoint = $"modelarmor.{locationId}.rep.googleapis.com",
+        };
+
+        // Create the client.
```

Comment:
We avoid this kind of comments because they describe what's evident and just add noise in the sample.
Here and everywhere.